src/FindApprenticeshipTraining/SFA.DAS.FindApprenticeshipTraining.Api.UnitTests/Controllers/AcademicYears/WhenQueryingAcademicYearsLatest.cs

```diff
@@ -0,0 +1,44 @@
+﻿using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.FindApprenticeshipTraining.Api.Controllers;
+using SFA.DAS.FindApprenticeshipTraining.Application.AcademicYears.Queries.GetLatest;
+using SFA.DAS.Testing.AutoFixture;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.FindApprenticeshipTraining.Api.UnitTests.Controllers.AcademicYears;
+public sealed class WhenQueryingAcademicYearsLatest
+{
+
+    [Test]
+    [MoqAutoData]
+    public async Task Then_Passes_Query_To_Mediator_And_Returns_AcademicYearsLatest(
```

Comment:
Something about the way this test is named feels odd to me, possibly because there's no assertion that the mediator is called. Not gonna hold up the PR for it but something to look at again

Reply:
renamed, changed to use fluent assertions, and mediatr verify added